src/gluonts/model/deepar/_estimator.py

```diff
@@ -171,7 +171,13 @@ def __init__(
         self.use_feat_static_cat = use_feat_static_cat
         self.use_feat_static_real = use_feat_static_real
         self.cardinality = cardinality if use_feat_static_cat else [1]
-        self.embedding_dimension = embedding_dimension
+        self.embedding_dimension = (
+            embedding_dimension
+            if embedding_dimension is not None
+            else [min(50, (cat + 1) // 2) for cat in cardinality]
+            if cardinality is not None
+            else None
```

Comment:
I think these two should be removed, since `embedding_dimension` is required by the network (it cannot be `None`)